.ci/wercker.sh

```diff
@@ -61,7 +61,7 @@ no-error-xwiki)
   git checkout i5812-rename-util
   mvn -f xwiki-commons-tools/xwiki-commons-tool-verification-resources/pom.xml \
     install -DskipTests -Dcheckstyle.version=${CS_POM_VERSION}
-  mvn -e test-compile checkstyle:check -Dcheckstyle.version=${CS_POM_VERSION}
+  mvn -e test-compile checkstyle:check@default -Dcheckstyle.version=${CS_POM_VERSION}
```

Comment:
Just a question. What does `@default` specifically do and why did it cause violations to be missed without it?

Reply:
@default enable the plugin execution id associated to https://github.com/xwiki/xwiki-commons/blob/master/xwiki-commons-tools/xwiki-commons-tool-verification-resources/src/main/resources/checkstyle.xml. This is the file which contains most of (or all depending how you see it) the checkstyle rules defined for XWiki.

If you don't set anything only https://github.com/xwiki/xwiki-commons/blob/master/xwiki-commons-tools/xwiki-commons-tool-verification-resources/src/main/resources/checkstyle-blocker.xml is enabled, this one contains a few things which are not exactly the usual code style and could be seen as compilation error but for XWiki specific stuff.

I'm not a big fan of the complexity brought to the checkstyle setup in XWiki but that's the current situation.

The specific missing violation is a change to IllegalType which started failing when HashMap was extended. According to the changelog this has been done on purpose so it's not some unexpected regression, but since we usually get a nice warning from you guys in cases like this I was surprised and wondered if the XWiki was still building properly on your side :)

Reply:
really sad fact ... strange that I did not caught that during testing, I usually test validation on simple violation introduction to code before placing it to CI, but smth might changed of xwiki side ....

but we need to bring it back to actually do testing.